### ISSUE TITLE
element.remove() function is not working in Internet explorer

### ISSUE THREAD

**Issue by AshishMotanamGurunadham** [cerner/carbon-graphs#307]
**What part of Carbon does this bug correlate to?**
General graph logic:

* [x]  Graph attributes (axes, legends, regions...etc.)
* [ ]  Other (please specify): 

**Describe the bug**
- When we try to destroy one of the tooltip div element in Carbon-graphs, we are getting error like "**Object does not support property or method called remove**" because IE does not support **remove()** function for DOM elements.

**Expected behavior**
- div element should be removed during destroy without any errors in all browsers.

**Screenshots**
![image](https://user-images.githubusercontent.com/63670637/95136702-3db29000-072c-11eb-9899-d87500944f9f.png)

**Desktop (please complete the following information):**

-   Browser [internet explorer]
